lib/Optimizer/CodeGen/Passes.cpp

```diff
@@ -35,11 +35,25 @@ static void addIQMPipeline(OpPassManager &pm) {
   pm.addPass(createBasisConversionPass(options));
 }
 
+static void addIonQPipeline(OpPassManager &pm) {
+  using namespace cudaq::opt;
+  // TODO: Add IonQ's lower level gates: ms, gpi, gpi2
+  std::string basis[] = {
+      "h", "s", "t", "r1", "rx", "ry", "rz", "x", "y", "z", "x(1)",
```

Comment:
```suggestion
using namespace std::literals;
  std::string_view basis[] = {
      "h"sv, "s"sv, "t"sv, "r1"sv, "rx"sv, "ry"sv, "rz"sv, "x"sv, "y"sv, "z"sv, "x(1)"sv
```
would avoid unneeded temporaries being created. May need to tweak `options` as well?